.travis.yml

```diff
@@ -3,38 +3,54 @@ node_js:
 install:
-    - yarn install --frozen-lockfile
-    - yarn cy:verify
-    - yarn build
+  - git clone --single-branch --branch gh-pages "https://${GITHUB_USERNAME}:${GITHUB_PASSWORD}@github.com/${FEEDBACK_REPO}" feedback-ci
+  - yarn install --frozen-lockfile
+  - yarn cy:verify
+  - yarn test
+  - yarn start &
 script:
-    - PORT=8081 REACT_APP_DHIS2_BASE_URL=http://localhost:8080 REACT_APP_CYPRESS=true yarn start &
-    - yarn wait-on http-get://localhost:8081
-    - yarn wait-on http-get://localhost:8080
-    - CYPRESS_EXTERNAL_API=http://localhost:8080 CYPRESS_ROOT_URL=http://localhost:8081 yarn cy:e2e:run #--record --key $CYPRESS_KEY
-    - kill $(jobs -p) || true
+  - yarn wait-on http-get://localhost:8080
+  - yarn wait-on http-get://localhost:8081
+  - yarn node cypress/support/runner.js -b chrome
+after_script:
+  - cd feedback-ci/
+  - if [[ $TRAVIS_PULL_REQUEST_BRANCH != "" ]]; then BRANCH="${TRAVIS_PULL_REQUEST_BRANCH}"; else BRANCH="${TRAVIS_BRANCH}"; fi;
+  - GITHUB_PAGES_ROOT=$(http -a ${GITHUB_USERNAME}:${GITHUB_PASSWORD} https://api.github.com/repos/${FEEDBACK_REPO}/pages | jq ".html_url" | tr -d '"')
+  - GITHUB_PAGES_PATH=$(cat ../pages_path)
+  - echo "- Built ``${TRAVIS_REPO_SLUG} ${TRAVIS_COMMIT_MESSAGE}`` against branch ${BRANCH} on $(date). [Report](${GITHUB_PAGES_ROOT}${GITHUB_PAGES_PATH})" >> README.md
+  - git add . && git commit -m "Results for $TRAVIS_EVENT_TYPE on $TRAVIS_COMMIT"
+  - git fetch origin
+  - git rebase -s ours origin/gh-pages
+  - git push "https://${GITHUB_USERNAME}:${GITHUB_PASSWORD}@github.com/${FEEDBACK_REPO}" gh-pages -f
+  - EXISTING_COMMENT=$(http -a ${GITHUB_USERNAME}:${GITHUB_PASSWORD} https://api.github.com/repos/${TRAVIS_REPO_SLUG}/issues/${TRAVIS_PULL_REQUEST}/comments | jq '[.[] | select(.user.login == "EyeSeeTeaBotTest")][0].id')
+  - if [[ $TRAVIS_TEST_RESULT == 0 ]]; then MESSAGE_TITLE="Tests ran successfully :tada:"; else MESSAGE_TITLE="Tests failed with errors :rotating_light:"; fi;
+  - if [[ $TRAVIS_PULL_REQUEST != "false" && $EXISTING_COMMENT == "null" ]]; then http -a ${GITHUB_USERNAME}:${GITHUB_PASSWORD} POST https://api.github.com/repos/${TRAVIS_REPO_SLUG}/issues/${TRAVIS_PULL_REQUEST}/comments body="<h3>${MESSAGE_TITLE}</h3><br />Build commit ${TRAVIS_COMMIT_MESSAGE} with commit id ${TRAVIS_COMMIT} on branch ${BRANCH}<br /><br />Travis log available at ${TRAVIS_BUILD_WEB_URL}<br /><br />See cypress results at ${GITHUB_PAGES_ROOT}${GITHUB_PAGES_PATH}"; fi;
+  - if [[ $TRAVIS_PULL_REQUEST != "false" && $EXISTING_COMMENT != "null" ]]; then http -a ${GITHUB_USERNAME}:${GITHUB_PASSWORD} PATCH https://api.github.com/repos/${TRAVIS_REPO_SLUG}/issues/comments/${EXISTING_COMMENT} body="<h3>${MESSAGE_TITLE}</h3><br />Build commit ${TRAVIS_COMMIT_MESSAGE} with commit id ${TRAVIS_COMMIT} on branch ${BRANCH}<br /><br />Travis log available at ${TRAVIS_BUILD_WEB_URL}<br /><br />See cypress results at ${GITHUB_PAGES_ROOT}${GITHUB_PAGES_PATH}"; fi;
```

Comment:
The eyeseetea.github.io page does not exist, was it cleared?

Reply:
No, the root page for ``eyeseetea.github.io`` would publish a repo called ``EyeSeeTea/eyeseetea.github.io``.

In this case since the repo is ``EyeSeeTea/feedback-ci`` it is published at https://eyeseetea.github.io/feedback-ci.

Reply:
I meant the one linked in the comment of the PR returned a 404. But others are working, no problem.

Reply:
> I meant the one linked in the comment of the PR returned a 404. But others are working, no problem.

Ah I see, no that's because it was overwritten by a concurrent job, what I'm trying to leave solved.